modules/system-management/pages/management-commands.adoc

```diff
@@ -887,18 +887,20 @@ $ gadmin status
 +--------------------+-------------------------+-------------------------+
 ----
 
-You  can also check the status of each instance using the verbose flag : `gadmin status -v`  or `gadmin status --verbose`. This will show each machine's status. See example below
+You  can also check the status of each instance using the verbose flag : `gadmin status -v`  or `gadmin status --verbose`. This will show each machine's status along with additional information like *Process ID* and *Role*. See the updated example below:
```

Comment:
```suggestion
You  can also check the status of each instance using the verbose flag : `gadmin status -v`  or `gadmin status --verbose`. This will show each machine's status along with additional information like *Process ID* and *Role*. See the example below:
```